.circleci/main.yml

```diff
@@ -1414,15 +1417,25 @@ jobs:
       - run: #Note: --no-ansi https://github.com/python-poetry/poetry/issues/7184
           name: Install Python Dependencies with Poetry
           command: poetry --no-ansi install
-      - install-python-test-deps
-
-      # export PACH_PYTHON_ENTERPRISE_CODE=${ENT_ACT_CODE}
-      # export export PACH_PYTHON_AUTH_TOKEN=iamroot
       - run:
-          name: Run Tests
+          name: Start minikube
+          command: minikube start --vm-driver=docker --kubernetes-version=v1.19.0 --cpus=7 --memory=12g --wait=all
+      - run:
+          name: Install pach
           command: |
-            nohup pachctl port-forward &
-            poetry run pytest -vvv tests
+            helm install pachd ../etc/helm/pachyderm --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=LoadBalancer
+            kubectl create namespace enterprise --dry-run=client -o yaml | kubectl apply -f -
+            helm install pachd-enterprise ../etc/helm/pachyderm --namespace enterprise --set deployTarget=LOCAL,pachd.image.tag=${CIRCLE_SHA1},proxy.service.type=LoadBalancer,proxy.service.httpPort=81,pachd.enterpriseLicenseKey=${ENT_ACT_CODE}
```

Comment:
I think we can simplify this by removing the enterprise install and moving the `pachd.enterpriseLicenseKey=${ENT_ACT_CODE}` to the main pachyderm helm install